docs/experiments-plus/overrides.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Overrides
+sidebar_label: Overrides
+slug: /experiments-plus/overrides
+---
+
+## Override group allocation for an Experiment
+
+During development, it can be useful to explicity state which experiment group a user should fall into. For example, if a developer is testing that each of the experiment conditions work as expected, they may want to force themselves into each of the different conditions to test.
+
+Overrides can be added based on Feature Gates and Segments, and will allow you to create rules that force all users that pass a given Feature Gate or Segment to be allocated to a given group. When fetching parameters for an experiment, if the user matches any of the overrides, the overriden result will be returned immediately.
+
```

Comment:
I think we should call out that data for users in overrides will not be included in pulse, due to bias, and highlight that here. It's being asked A LOT.